clicommand/agent_start.go

```diff
@@ -939,6 +950,30 @@ var AgentStartCommand = cli.Command{
 			l.Info("Agents will disconnect after %d seconds of inactivity", agentConf.DisconnectAfterIdleTimeout)
 		}
 
+		if len(cfg.AllowedRepositories) > 0 {
+			agentConf.AllowedRepositories = []*regexp.Regexp{}
```

Comment:
you could probably change this for 
```suggestion
			agentConf.AllowedRepositories = make([]*regexp.Regexp, 0, len(cfg.AllowedRepositories)
```
to save a couple of slice allocations, but it's not a big deal - the length is likely to be pretty short most of the time.